up_client_zenoh/examples/common_uuri.py

```diff
@@ -0,0 +1,63 @@
+"""
+SPDX-FileCopyrightText: 2024 Contributors to the Eclipse Foundation
+
+See the NOTICE file(s) distributed with this work for additional
+information regarding copyright ownership.
+
+This program and the accompanying materials are made available under the
+terms of the Apache License Version 2.0 which is available at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+SPDX-License-Identifier: Apache-2.0
+"""
+import json
+import logging
+import zenoh
+from enum import Enum
+from typing import Optional
+from uprotocol.proto.uri_pb2 import UAuthority
+from uprotocol.proto.uri_pb2 import UEntity
+from uprotocol.proto.uri_pb2 import UResource
+from uprotocol.uri.factory.uresource_builder import UResourceBuilder
+
+# Configure the logging
+logging.basicConfig(level=logging.DEBUG, format='%(asctime)s - %(levelname)s - %(message)s')
+
+
+class ExampleType(Enum):
+    PUBLISHER = "publisher"
+    SUBSCRIBER = "subscriber"
+    RPC_SERVER = "rpc_server"
+    RPC_CLIENT = "rpc_client"
+
+
+def authority() -> UAuthority:
+    return UAuthority(name="auth_name", id=bytes([1, 2, 3, 4]))
+
+
+def entity(example_type: ExampleType) -> UEntity:
+    mapping = {ExampleType.PUBLISHER : ("publisher", 1), ExampleType.SUBSCRIBER: ("subscriber", 2),
+               ExampleType.RPC_SERVER: ("rpc_server", 3), ExampleType.RPC_CLIENT: ("rpc_client", 4)}
+    name, id = mapping[example_type]
+    return UEntity(name=name, id=1, version_major=id)
+
+
+def pub_resource() -> UResource:
+    return UResource(name="door", instance="front_left", message="Door", id=5678)
+
+
+def rpc_resource() -> UResource:
+    return UResourceBuilder.for_rpc_request("getTime", 5678)
+
+
+def get_zenoh_config():
+    zenoh_ip = "192.168.29.79"  # zenoh router ip
```

Comment:
Maybe this should become an argument from users?

Reply:
It could also become an argument, but since this is just an example, I've hardcoded the IP and port.

Reply:
I mean we don't need the Zenoh router to run the examples.
I would suggest we can have either way:
1. Remove (or comment) the hardcoded IP and port here. Let's use the default config.
2. Ask users to run Zenoh router and also modify the IP and port here in the README.

Reply:
Understood, I've added another method for the default configuration. All the examples now use this default config. Users can refer to an alternative method to set the configuration if the zenoh router is running.